release/ray_release/test_automation/state_machine.py

```diff
@@ -57,10 +92,63 @@ def _move_hook(self, from_state: TestState, to_state: TestState) -> None:
         Action performed when test transitions to a different state. This is where we do
         things like creating and closing github issues, trigger bisects, etc.
         """
-        if (from_state, to_state) == (TestState.PASSING, TestState.FAILING):
+        if (from_state, to_state) == (TestState.PASSING, TestState.CONSITENTLY_FAILING):
```

Comment:
since `(from_state, to_state)` is repeated, maybe:

````
change = from_state, to_state

if change == TestState.PASSING, TestState.CONSITENTLY_FAILING:
  ...
elif change == TestState.FAILING, TestState.CONSITENTLY_FAILING:
  ...

````

Reply:
smart!